definitions/npm/nprogress_v0.2.0/flow_v0.25.x-/nprogress_v0.2.0.js

```diff
@@ -0,0 +1,102 @@
+declare module "nprogress" {
+  declare type NProgressStatic = {
+    /**
+     * Shows the progress bar and begins trickling progress.
+     * @returns {NProgressConfigureOptions} The current NProgress object, useful for chaining.
```

Comment:
All your `@returns` annotations says `NProgressConfigureOptions` with is confusing since it the options type.   
I guess it's `NProgressureStatic` instead?